### PR TITLE
Technical Advisories

### PR DESCRIPTION
**Pull request by premkumr**
@netlify /preview/releases/techadvisories/

### CODE REVIEW

**Comment by premkumr** [comment]
@netlify /preview/releases/techadvisories/

File: docs/content/preview/releases/techadvisories/_index.md [hunk 1]
```diff
@@ -12,32 +12,14 @@ cascade:
...
-
-- A brief overview of the problem
-- Products/versions affected
-- Recommended actions or workarounds
-- Links to additional resources
-
-We strongly recommend regularly reviewing this page and taking appropriate measures outlined in the advisories to mitigate potential risks and disruptions.
-
-Our team of experts continuously monitors for emerging threats and technical concerns. New advisories will be posted here as soon as they become available.
+We strongly recommend regularly reviewing this page and taking appropriate measures outlined in the advisories to mitigate potential risks and disruptions. Our team continuously monitors for emerging threats and technical concerns. New advisories will be posted here as soon as they become available.
```
Comment (aishwarya24) [inline docs/content/preview/releases/techadvisories/_index.md:15]:
```suggestion
It is strongly recommended that you regularly review this page and take appropriate measures outlined in the advisories to mitigate potential risks and disruptions. Yugabyte continuously monitors for emerging threats and technical concerns.
```

File: docs/content/preview/releases/techadvisories/_index.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+Here you will find important notifications and updates regarding product issues, security vulnerabilities, and other technical matters that may impact our products and services. We understand that keeping your systems secure and running smoothly is of utmost importance. This page serves as a central resource to ensure you have access to the latest technical advisories issued by our team.
+
+We strongly recommend regularly reviewing this page and taking appropriate measures outlined in the advisories to mitigate potential risks and disruptions. Our team continuously monitors for emerging threats and technical concerns. New advisories will be posted here as soon as they become available.
+
+## List of advisories
+
+| Advisory&nbsp;Number |    Synopsis     | Product  | Affected Versions | Publish&nbsp;Date |
+| :------- | :-------------- | :------- | :---------------- | :----------- |
+| {{<ta 21297>}}| Missing writes during Batch Execution in a transaction | [YSQL](./) | [All](/preview/releases/ybdb-releases/)            | Mar-25-2024  |
```
Comment (aishwarya24) [inline docs/content/preview/releases/techadvisories/_index.md:21]:
```suggestion
| {{<ta 21297>}}| Missing writes during Batch Execution in a transaction | [YSQL](../../api/ysql/) | [All](../ybdb-releases/)            | March 25, 2024  |
```

File: docs/content/preview/releases/techadvisories/_index.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+linkTitle: Technical Advisories
+description: List of technical advisories
+image: /icons/triangle-exclamation-thin.svg
+type: indexpage
+showRightNav: true
+cascade:
+  unversioned: true
+---
+
+Here you will find important notifications and updates regarding product issues, security vulnerabilities, and other technical matters that may impact our products and services. We understand that keeping your systems secure and running smoothly is of utmost importance. This page serves as a central resource to ensure you have access to the latest technical advisories issued by our team.
```
Comment (aishwarya24) [inline docs/content/preview/releases/techadvisories/_index.md:13]:
```suggestion
Technical Advisories include important notifications and updates regarding product issues, security vulnerabilities, and other technical matters that may impact Yugabyte products and services. We understand that keeping your systems secure and running smoothly is of utmost importance. 
This page serves as a central resource to ensure you have access to the latest technical advisories issued by Yugabyte.
```

**Review by aishwarya24 — COMMENTED** [review]
landing page comments

File: docs/content/preview/releases/techadvisories/ta-21297.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+- You use some form of write "batch statement" (for example, multiple write statements into the same message) specifically as the first message/statement in the transaction; and a transaction conflict occurs for any but the first internal statement in that batch statement.
+
+## Mitigation
+
+The workaround is to add a read statement (for example, `SELECT 1`) as the first statement in each transaction.
+
+You can also disable the transaction retries according to the installed release as follows:
+
+- Versions prior to {{<release "2.18.5.1, 2.18.6.0, 2.20.1.1,2.20.2.0, 2.21.0.0">}}.
```
Comment (aishwarya24) [inline docs/content/preview/releases/techadvisories/ta-21297.md:37]:
```suggestion
- Versions prior to {{<release "2.18.5.1, 2.18.6.0, 2.20.1.1, 2.20.2.0, 2.21.0.0">}}.
```

File: docs/content/preview/releases/techadvisories/ta-21297.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+    Set the following flags:
+
+    ```sql
+    ysql_max_read_restart_attempts = 0
+    ysql_max_write_restart_attempts = 0
+    ```
+
+- Versions starting from {{<release "2.18.5.1">}}, {{<release "2.18.6.0">}}, {{<release "2.20.1.1">}}, {{<release "2.20.2.0">}}, and {{<release "2.21.0.0">}}.
```
Comment (aishwarya24) [inline docs/content/preview/releases/techadvisories/ta-21297.md:46]:
```suggestion
- Versions starting from {{<release "2.18.5.1, 2.18.6.0, 2.20.1.1, 2.20.2.0, and 2.21.0.0">}}
```

Comment (aishwarya24) [reply docs/content/preview/releases/techadvisories/ta-21297.md:46]: @premkumr - is this correct as well? To include "and" inside or should it be:
- Versions starting from {{<release "2.18.5.1, 2.18.6.0, 2.20.1.1, 2.20.2.0">}}, and {{<release "2.21.0.0">}}

**Review by aishwarya24 — APPROVED** [review]
LGTM

File: docs/content/preview/releases/techadvisories/ta-21297.md (hunk 5 above)
Comment (premkumr) [reply docs/content/preview/releases/techadvisories/ta-21297.md]: It does not support and. For now , we can use it the way you way suggested. We can discuss this later.

**Pull request merged**